charms/docker/compose.py

```diff
@@ -47,6 +47,19 @@ def kill(self, service=None):
             cmd = "docker-compose kill"
         self.run(cmd)
 
+    def rm(self, service=None):
+        '''
+        Convenience method that wraps `docker-compose rm`
+
+        usage: c.rm('nginx') to remove the 'nginx' service from the
```

Comment:
I see that the wording is copied this from above replacing 'run' with 'remove'. This comment could be interpreted that the rm edits the docker-compose.yml file.